manubot/cite/tests/test_curie.py

```diff
@@ -1,69 +1,67 @@
 import pytest
 
-from ..curie import curie_to_url, get_namespaces, get_prefix_to_namespace
+from ..citekey import CiteKey
+from ..curie import Handler_CURIE, curie_to_url, get_bioregistry, get_prefix_to_registry
 
 
-def test_get_namespaces_with_compile_patterns():
+@pytest.mark.xfail(reason="https://github.com/biopragmatics/bioregistry/issues/242")
+def test_registry_patterns():
     """
-    To see printed output when this test passes, run:
-
-    ```shell
-    # show regexes that do not match the sampleId
-    pytest --capture=no --verbose manubot/cite/tests/test_curie.py
-    ```
+    https://github.com/biopragmatics/bioregistry/issues/242
     """
-    namespaces = get_namespaces(compile_patterns=True)
-    assert isinstance(namespaces, list)
-    for namespace in namespaces:
-        # ensure prefix field exists
-        assert namespace["prefix"]
-        # check whether compiled pattern matches example identifier
-        # do not fail when no match, since this is an upstream issue
-        # https://github.com/identifiers-org/identifiers-org.github.io/issues/99
-        compact_id = namespace["sampleId"]
-        if namespace["namespaceEmbeddedInLui"]:
-            compact_id = f"{namespace['curiePrefix']}:{compact_id}"
-        match = namespace["compiled_pattern"].fullmatch(compact_id)
-        if not match:
-            print(
-                f"{namespace['prefix']} regex "
-                f"{namespace['compiled_pattern'].pattern} "
-                f"does not match {compact_id}"
-            )
-        if namespace["prefix"] != namespace["curiePrefix"].lower():
-            print(
-                f"{namespace['prefix']} identifiers use "
-                f"curiePrefix {namespace['curiePrefix']}"
-            )
+    bioregistry = get_bioregistry(compile_patterns=True)
+    assert isinstance(bioregistry, list)
+    reports = list()
+    for registry in bioregistry:
```

Comment:
I would call each entry in the bioregistry a resource, not a registry.

Reply:
Ah i came to this conclusion independently and updated in https://github.com/manubot/manubot/pull/306/commits/6db6ae6b47652086bc4407d6cca8ee3ba1c677b4. Nice to see that we picked the same term "resource"

Reply:
Here's the glossary of all of the terms I'm using: https://cthoyt.com/2021/10/07/biopragmatics-glossary.html

Will port it to a more neutral location that isn't my blog in the near future

Reply:
What about that slide you showed with definitions of things like thesuarus and vocabulary... is that somewhere online?

Reply:
https://bit.ly/WPCI2021

I will be putting together a summary of the whole workshop by the end of this week hopefully